### DISCUSSION TITLE
I compile the VS ITel and Anura properly ,but the result of the analysis is like this.

### DISCUSSION THREAD

**Discussion by girmameaza12** [Anura3D/Anura3D_OpenSource#136] [Q&A] [answered]
![Capture](https://github.com/Anura3D/Anura3D_OpenSource/assets/141491544/3ffbe54e-1bb1-421b-9aa3-fa518eca6e3f)

**Comment by FrancescaCeccato** [comment, marked as answer]
is the dll inside the same folder of the code you are running?

**Comment by girmameaza12** [comment]
Thanks 🙏🏻 it's working now.i reinstall it .
